test/iam.ts

```diff
@@ -144,20 +141,14 @@ describe('IAM', () => {
   });
 
   describe('testPermissions', () => {
-    it('should throw an error if permissions are missing', () => {
```

Comment:
again, why deleting a test?

Reply:
same scenario as setPolicy function.
https://github.com/googleapis/nodejs-pubsub/blob/master/src/iam.ts#L313

Reply:
fix the changes as suggested on another PR
https://github.com/googleapis/nodejs-pubsub/pull/502